en/publish/publish-wechatgame-sub-domain.md

```diff
@@ -131,35 +132,35 @@ Since the code and resources of the WeChat open data domain cannot be shared wit
             this.tex.handleLoadedTexture();
             this.display.spriteFrame = new cc.SpriteFrame(this.tex);
         },
-        Update () {
+        update () {
```

Comment:
There are still some capitalized code that has not been changed back, please refer to your previous changes:
![image](https://user-images.githubusercontent.com/19702899/49422284-45bae500-f7ce-11e8-8ba5-0378eb08570e.png)